boards/arm/nrf54h20pdk_nrf54h20/nrf54h20pdk_nrf54h20_cpurad.dts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nordic/nrf54h20_eng0_cpurad.dtsi>
+#include "nrf54h20pdk_nrf54h20-memory_map.dtsi"
+#include "nrf54h20pdk_nrf54h20_cpurad-pinctrl.dtsi"
+
+/ {
+	compatible = "nordic,nrf54h20pdk_nrf54h20-cpurad";
+	model = "Nordic nRF54H20 PDK nRF54H20 Radio MCU";
+
+	chosen {
+		zephyr,console = &uart136;
+		zephyr,code-partition = &cpurad_slot0_partition;
+		zephyr,flash = &mram1x;
+		zephyr,sram = &cpurad_ram0;
+	};
+};
+
+&ram3x_dma_region {
+	status = "okay";
+};
+
+&cpurad_dma_region {
+	status = "okay";
+};
+
+&cpurad_rx_partitions {
+	status = "okay";
+};
+
+&grtc {
+	status = "okay";
+	child-owned-channels = <8 9 10 11 12>;
+	interrupts = <109 NRF_DEFAULT_IRQ_PRIORITY>,
+		     <108 NRF_DEFAULT_IRQ_PRIORITY>;
+	nonsecure-channels = <8 9 10 11 12>;
+	owned-channels = <7 8 9 10 11 12 13 14>;
+};
+
+&uart135 {
+	pinctrl-0 = <&uart135_default>;
+	pinctrl-1 = <&uart135_sleep>;
+	pinctrl-names = "default", "sleep";
+};
```

Comment:
Why this node is defined here if its status is "disabled"?

Reply:
Ok, now I'm confused. I see for the nrf54h20pdk_nrf54h20 both cpuapp and cpurad set `status = "okay"` for `&uart136` and both set `pinctrl` for `&uart135` with `status = "disabled"`. I expected cpuapp would set `status = "okay"` and `pinctrl` properties for one of the UART instances, while cpurad would do the same for the other one.
How the resources access conflict is resolved now?

Reply:
Right. In a multi-core application, cpurad would have to disable `uart136` and enable `uart135` to avoid conflicts.

I'm not sure which default configuration would be preferred in Zephyr:

1. Both cores use the same UART, so that if you run a basic sample on either one, it will output over the same pins.
2. Both cores use different UARTs, to support multi-core samples without extra overlays.

Reply:
In nRF5340 DK, we use separate pins (hence, separate VCOMs) for UART consoles on the Application and Network cores. We should probably use a similar scheme here and consequently use separate UARTs for these two cores.

Reply:
Changed to use separate UARTs and pins on Application and Radio cores.